node_modules/col-users/lib/rest.js

```diff
@@ -41,6 +41,26 @@ Collabosphere.apiRouter.get('/users/me', function(req, res) {
 });
 
 /*!
+ * Get user of current course by id
+ */
+Collabosphere.apiRouter.get('/users/:id', function(req, res) {
+  // Get user by id
+  var userId = req.params.id;
+  // Only admins can access non-self user data
+  if (userId != req.ctx.user.id && !req.ctx.user.is_admin) {
+    return res.status(401).send('Unauthorized API request');
+  }
```

Comment:
@johncrossman Is it the case that this endpoint will be totally inaccessible to non-admins? It seems like we might want to return at least a subset of properties that users have made public.

Also, convention would be for authentication checks to take place in `api.js`, not `rest.js`. We might want a new UserAPI method that would retrieve the User object for ID, validate that the user exists in ctx.course (otherwise returning 400), and filter out any properties that should be hidden from non-admins.